name.abuchen.portfolio.ui/src/name/abuchen/portfolio/ui/views/AttributeSettingsPane.java

```diff
@@ -215,11 +215,19 @@ private static void createColorView(Composite parent, IStylingEngine stylingEngi
             // before
             stylingEngine.style(gpColors);
 
-            int width = SWTHelper.widest(positiveExceeded.getFirst(), negativelyExceeded.getSecond());
+            // Compute the widest element width in the group using
+            int width = SWTHelper.widest(positiveExceeded.getFirst(), negativelyExceeded.getFirst(), button,
+                            resetButton);
```

Comment:
"button" and "resetButton" are displayed next to each other.

I do not unterstand whether it makes to determine their size separately. Even if one of the buttons is the widest element, it does not help because both are displayed next to each other.